packages/thyra/core/cmake/Dependencies.cmake

```diff
@@ -1,4 +1,4 @@
 TRIBITS_PACKAGE_DEFINE_DEPENDENCIES(
   LIB_REQUIRED_PACKAGES TeuchosCore TeuchosParameterList TeuchosComm TeuchosNumerics
-    RTOp
+    RTOp Thyra
```

Comment:
The fact that TriBITS allowed this to configure shows that there is a missing check in TriBITS that did not correctly detect a circular dependency in this case.